ZuluContent/Items/TreasureChests/TreasureChestLevel1.cs

```diff
@@ -53,7 +53,7 @@ public TreasureChestLevel1()
 			Movable = false;
 
 			TrapType = TrapType.DartTrap;
-			TrapPower = m_Level * Utility.Random( 1, 25 );
+			TrapStrength = m_Level * Utility.Random( 1, 25 );
```

Comment:
Should we try to standardize on using RandomMinMax rather or is there a big difference here?

Reply:
`RandomMinMax` should be inclusive which makes more logical sense to me when I read the code